BEST_PRACTICES.md

```diff
@@ -22,29 +41,23 @@ The recommended version of Xcode (and Swift) for all purposes is the current ava
 
 Enable warnings by adding `-Weverything` to your Build Settings under "Other Compiler Flags". If you need to ignore a specific warning you can use [Clang's pragma feature](http://clang.llvm.org/docs/UsersManual.html#controlling-diagnostics-via-pragmas) or add `-Wno-warning-to-be-disabled` (for example `-Wno-gnu-conditional-omitted-operand`).
 
-## Deployment
+## Versioning
 
-### Semantic Versioning
-
-We support [semantic versioning](http://semver.org/), and it's important that minor releases are backwards compatible otherwise don't feel shy to make it a major release.
+To version our open source projects we use [semantic versioning](http://semver.org/), and it's important that minor releases are backwards compatible otherwise don't feel shy to make it a major release.
 
 When making backwards compatible changes, flag your old APIs as deprecated like this:
 
-**Objective-C**:
-```objc
-- (NSInteger)objectWithIndex:(NSInteger)bar __attribute__((deprecated("Use objectAtIndex: instead")));
-```
-
-**Swift**:
 ```swift
 @available(*, deprecated=4.3.0, message="Use `objectAt(index index: Int)` instead") public func objectAtIndex(index: Int)
 ```
 
+When it comes to apps, patch releases are bug fixes, minor releases are small new features and major releases are re-designs or big features.
+
 ## Comments
 
 When they are needed, comments should be used to explain **why** a particular piece of code does something instead of **what**. Any comments that are used must be kept up-to-date or deleted.
 
-Block comments should generally be avoided, as code should be as self-documenting as possible, with only the need for intermittent, few-line explanations. This does not apply to those comments used to generate documentation.
+Comments should generally be avoided, as code should be as self-documenting as possible, with only the need for intermittent, few-line explanations. This does not apply to those comments used to generate documentation.
```

Comment:
I think it's fine to keep the `code should be as self-documenting as possible` because that's the goal.